abis/linux/signal.h

```diff
@@ -400,7 +400,6 @@ typedef struct __ucontext {
 	struct ucontext	*uc_link;
 	stack_t uc_stack;
 	sigset_t uc_sigmask;
-	uint8_t  __unused[1024 / 8 - sizeof(sigset_t)];
```

Comment:
Is this not an ABI break? I guess my real question is: is `sizeof(sigset_t)` always 128? On all architectures?

Reply:
On Linux, yes currently it's always 128:
https://github.com/managarm/mlibc/blob/0f7479f4645d36897253235c40b134f49ba989f6/abis/linux/signal.h#L123-L125

Perhaps I should suppress the check here just to be safe? Or add some kind of assert?

Reply:
Maybe add a static assert to check the alignment of `struct __ucontext`, since that seems to be the purpose of the padding.

Reply:
I've kept it but suppressed it with a pragma